docs/rdataretriever.rst

```diff
@@ -56,3 +263,90 @@ To get citation information for the ``rdataretriever`` in R use ``citation(packa
 
 
 .. _Data Retriever: http://data-retriever.org
+
+
+RDataRetriever with PostgreSQL
+==============================
+
+RDataRetriever supports different DBMS for storing of downloaded datasets through the Retriever R API. It supports PostgreSQL, MySQL, SQLite and MS Access.
+
+
+Installation of PostgreSQL in Ubuntu
```

Comment:
This is really useful information, but I'm not sure if information on installing postgres into Ubuntu is within scope for our documentation. It seems like if we're going to include this we'd also want to include other DBMSs and OSs. What do you think @pranita-s & @henrykironde?

Reply:
@ethanwhite @henrykironde I can include information for other DBMSs and OSs too. If there are any content enhancements needed, taking current information for "Installation for PostgreSQL" into consideration, it will be great to know !

Reply:
I don't think I see anything that needs to be added in general, but I also think it would be find to link to other installation guides and just mention anything that is specific to the retriever that needs to be done.